src/uniprotkb/components/results/__tests__/__snapshots__/UniProtKBCard.spec.tsx.snap

```diff
@@ -108,7 +108,7 @@ exports[`UniProtKBCard component should render 1`] = `
         class="card__link"
       >
         <a
-          href="/uniprotkb/P21802#sequence"
+          href="/uniprotkb/P21802#Sequence"
```

Comment:
I think this is incorrect, it should be the lowercase id used and it looks like it's using the name